src/validation/ValidationExecutor.ts

```diff
@@ -45,13 +45,14 @@ export class ValidationExecutor {
      */
     if (!this.metadataStorage.hasValidationMetaData && this.validatorOptions?.enableDebugMessages === true) {
       console.warn(
-        `No metadata found. There is more than once class-validator version installed probably. You need to flatten your dependencies.`
+        `No metadata found. There is more than one class-validator version installed probably. You need to flatten your dependencies.`
```

Comment:
This change seems to no belong in this PR, and there is another PR for this already https://github.com/typestack/class-validator/pull/1127